e2e/txpool_test.go

```diff
@@ -400,68 +400,6 @@ func TestTxPool_StressAddition(t *testing.T) {
 	}
 }
 
-func TestInvalidTransactionRecover(t *testing.T) {
```

Comment:
This test is not valid anymore. Because now transaction will never go inside txPool if gas > latestBlockGasLimit.

Reply:
Please link the unit test(s) that cover this functionality in the new TxPool implementation 🙏